js/scripts/conversation_loaders/village_conversation_loader.js

```diff
@@ -182,6 +182,51 @@ const blacksmithConversationLoader = () => {
             new DialogBubble(bs, "mmhm...", true),
             // 9
             new DialogBubble(bs, "Gonna need more than that.", true)
+        ], merchantScared: [
```

Comment:
I like the addition of the merchant after the village attack. Now there's actually a reason to collect runes :)

Reply:
Yup! Was the easiest fix to that problem I could think of. Now dying and respawning at village will actually be useful since you keep your items and runes upon death and can use your runes before going back to fight! It is helpful for the game-play loop similar to how in say Elden Ring when you want to grind and farm beasts for runes where you would just reset at the grace after defeating them you can just collect runes and upon death use them before fighting again. @tclaar at one point you offered to take on setting the respawn for Chad upon death to the current zone, I think we should not do that now for the reasoning I listed above.